index.html

```diff
@@ -9,261 +9,392 @@
+                     <img class="images" src="images/ic_skills.svg" alt="#">
+                     
+                 </div>
+                 <div class="langs-img-dsk display-none">
+                    <img class="images" src="images/image_dots_with_teal.svg" alt="#">
+
+                 </div>
+             </div>
+
+             <div class="about-me-dsk display-none">
+                <h3>About</h3>
+                <P>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</P>
+        
+                <button name="view-project" type="button">
+                    <span class="project-view-btn">Get my CV</span>
+                    
+                    <span><img class="cv-btn cursor" src="images/ic_download.png" alt="union"></span>
+                </button>
             </div>
-            <div class="frameworks">
-                <div class="content-list">
-                    <h3>Frameworks</h3>
-                <ul>
-                    <li>Bootstrap</li>
-                    <li>Ruby on rails</li>
-                    <li>RSpec</li>
-                    <li>Capibara</li>
-                    <li>Selenium</li>
-                </ul>
-    
+             
+         </section>
+         <footer>
+            <div class="footer-container">
+                <div class="header-img-left display-none">
+                    <img src="images/image-geometry_6.png" alt="image">
                 </div>
-                <img class="images" src="images/Group_18.png" alt="#">
-    
-            </div>
-            <div class="skills">
-                <div class="content-list">
-                    <h3>Skills</h3>
-                <ul>
-                    <li>Codekit</li>
-                    <li>Codepen</li>
-                    <li>Github</li>
-                    <li>GitLAB</li>
-                    <li>Terminal</li>
-                </ul>
+                <div class="contact">
+                    <div class="contact-description">
+                        <h3>Get started</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+                    </div>
+                    <form action="https://formspree.io/f/xbjwgkqr" method="post">
+                        <input type="text" name="name" class="name" placeholder="Your name" maxlength="30" required>
+                        <input type="email" name="email" class="email" placeholder="Your email" required>
+                        <textarea name="message" class="msg" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        <button class="btn-collabo-desk btn-collabo" type="submit" name="submit">Start collaboration</button>
+        
+                    </form>
```

Comment:
- [ ] As requested by the previous reviewer, pleas amke sure to add some padding for every input to be like the design in Figma: 
<img width="323" alt="Screenshot 2022-02-25 at 13 08 57" src="https://user-images.githubusercontent.com/77038610/155687477-b9c61605-0da4-4c31-a9d4-34aec87a5053.png">

Reply:
Hi, I tried this but it does not work, do you have an idea?

Reply:
input[type="text"], [type="email"]::placeholder {  
                  
  padding-left: 10px;
  padding-top: 0px; 
} 
textarea::placeholder {  
                  
  padding: 15px; 
} 



I used this code, hope it is okay

Reply:
![screen-reviewer](https://user-images.githubusercontent.com/92704732/155693991-b14d5961-cb9c-4126-a746-4b9e769d2476.png)

Reply:
Hey, I can see that you have already solved the issue 👍🏻

Reply:
It looks great now ☀️

Reply:
Okay

Reply:
I have submitted, thanks